storm-client/src/jvm/org/apache/storm/Config.java

```diff
@@ -1676,6 +1678,18 @@
     //DO NOT CHANGE UNLESS WE ADD IN STATE NOT STORED IN THE PARENT CLASS
     private static final long serialVersionUID = -1550278723792864455L;
 
+    /**
+     * Specify the trigger policy for the window.
+     */
+    @IsType(type = TriggerPolicy.class)
+    public static final String TOPOLOGY_BOLTS_WINDOW_TRIGGER_POLICY = "topology.bolts.window.trigger.policy";
```

Comment:
Please move these to be next to the other window-related properties, such as sliding count.